android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -648,4 +652,61 @@ public JSONArray getSecondaryValues(JSONObject jsonObject, String type) {
 
         return value;
     }
+
+    public static JSONArray fields(JSONObject jsonForm) {
```

Comment:
Allow passing the step as a variable, or rename the method

Reply:
Done.